ingester/src/persist/context.rs

```diff
@@ -193,24 +195,31 @@ impl Context {
     /// diverged from the cached sort key in `self`, indicating concurrent
     /// persist jobs that update the sort key for the same [`PartitionData`]
     /// running in this ingester process.
-    pub(super) async fn set_partition_sort_key(&mut self, new_sort_key: SortKey) {
+    pub(super) async fn set_partition_sort_key(
+        &mut self,
+        new_sort_key: SortKey,
+        new_sort_key_ids: Option<SortedColumnSet>,
```

Comment:
I guess this will become:

```suggestion
        new_sort_key_ids: SortedColumnSet,
```

When we remove the options in https://github.com/influxdata/influxdb_iox/pull/8556/files#r1312789990 :+1:

Reply:
Done